drivers/scale_host.go

```diff
@@ -88,15 +101,26 @@ func (s *ScaleHostDriver) Execute(conf interface{}, apiClient *client.RancherCli
 		return http.StatusInternalServerError, errors.Wrap(err, "Couldn't unmarshal config")
 	}
 
+	hostTemplate, err := apiClient.HostTemplate.ById(config.HostTemplateID)
+	if err != nil {
+		return http.StatusBadRequest, fmt.Errorf("Error in get HostTemplate")
```

Comment:
This message will be returned to the user, so it needs to be more readable.
```
"Error getting hostTemplate resource"
```
Also, you should log the error here via logrus.Errorf()